tours/src/main/res/layout/cell_tour_details_stop.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:paddingTop="@dimen/marginOneHalf"
+    android:paddingBottom="@dimen/marginOneHalf">
+
+    <ImageView
+        android:id="@+id/image"
+        android:layout_width="72dp"
+        android:layout_height="45dp"
+        android:contentDescription="@null" />
+
+    <TextView
+        android:id="@+id/tourNumber"
+        android:layout_width="20dp"
+        android:layout_height="22dp"
+        app:layout_constraintStart_toEndOf="@id/image"
+        android:layout_marginStart="@dimen/marginOneHalf"
+        android:layout_marginTop="3dp"
+        />
+
+    <LinearLayout
```

Comment:
dont nest another type of view here. we can use `ConstraintLayout` to flatten the hierarchy. if you need help I can show you.

Reply:
we can use a `Barrier` to align the two text to the right of the `tourNumber` if needed.

Reply:
or even just simple `_toEndOf`

Reply:
Its mostly so that we can get the proper spacings in. I can modify it in the branch it is supposed to be done in.